app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -64,4 +66,192 @@ class SmokeTest {
             }
         }
     }
+
+    @Test
+    fun verifyPageMainMenuItemsListPotraitMode() {
+        val defaultWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+
+        homeScreen {
+            navigationToolbar {
+            }.enterURLAndEnterToBrowser(defaultWebPage.url) {
+            }.openThreeDotMenu {
+                verifyThreeDotMainMenuItems()
+            }.openHistory {
+                verifyTestPageUrl(defaultWebPage.url)
+            }.goBackToBrowser {
+            }.openThreeDotMenu {
+            }.openBookmarks {
+                verifyBookmarksMenuView()
+                verifyEmptyBookmarksList()
+            }.goBackToBrowser {
+            }.openThreeDotMenu {
+            }.openSettings {
+                verifySettingsView()
+            }.goBackToBrowser {
+            }.openThreeDotMenu {
+            }.openFindInPage {
+                verifyFindInPageSearchBarItems()
+            }.closeFindInPage {
+            }.openThreeDotMenu {
+            }.addToFirefoxHome {
+                verifySnackBarText("Added to top sites!")
+            }.openTabDrawer {
+            }.openHomeScreen {
+                verifyExistingTopSitesTabs(defaultWebPage.title)
+            }.openTabDrawer {
+            }.openTab(defaultWebPage.title) {
+            }.openThreeDotMenu {
+            }.openAddToHomeScreen {
+                verifyShortcutNameField(defaultWebPage.title)
+                clickAddShortcutButton()
+                clickAddAutomaticallyButton()
+                verifyShortcutIcon()
+            }.openHomeScreenShortcut(defaultWebPage.title) {
+            }.openThreeDotMenu {
+            }.bookmarkPage {
+                verifySnackBarText("Bookmark saved!")
+            }.openThreeDotMenu {
+                verifyEditBookmarkButton()
+            }.openEditBookmarks {
+                clickdeleteBookmarkButton()
+            }.confirmBookmarkFolderDeletionAndGoBackToBrowser {
+                verifySnackBarText("Deleted localhost")
+            }.openThreeDotMenu {
+            }.sharePage {
+                // verifyShareAppsLayout()
+                // verifyShareOverlay()
+            }.closeShareDialogReturnToPage {
+            }.openThreeDotMenu {
+            }.refreshPage {
+                // verifyPageContent(defaultWebPage.content)
+            }.openTabDrawer {
+                closeTabViaXButton(defaultWebPage.title)
+            }.openHomeScreen {
+
+                // PRIVATE BROWSER

```

Comment:
As we talked on Riot, yes, I think it make sense to split the test for simplicity

Reply:
So, 4 tests?

Reply:
okay, let's try that approach

Reply:
@isabelrios I will open other PR, this one got messed up

Reply:
yes @TejaswiKarasani, sounds good. Closing this one then. Thanks